lib/srv/db/postgres/sql/delete-user.sql

```diff
@@ -9,7 +9,7 @@ BEGIN
         RAISE NOTICE 'User has active connections';
     ELSE
         BEGIN
-            EXECUTE FORMAT('DROP USER %I', username);
+            EXECUTE FORMAT('DROP USER IF EXISTS %I', username);
```

Comment:
Thanks for the fix. Let's also apply this to `redshift-delete-user.sql`
https://docs.aws.amazon.com/redshift/latest/dg/r_DROP_USER.html